server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -2784,6 +2792,28 @@ public ComplianceStatusDTO getComplianceStatus(
         return this.translator.translate(status, ComplianceStatusDTO.class);
     }
 
+    @ApiOperation(notes = "Retrieves the System Purpose Compliance Status of a Consumer.", value =
+        "getSystemPurposeComplianceStatus")
+    @ApiResponses({ @ApiResponse(code = 404, message = "") })
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("{consumer_uuid}/purpose_compliance")
+    @Transactional
+    public SystemPurposeComplianceStatusDTO getSystemPurposeComplianceStatus(
```

Comment:
No idea if this is something we need, but maybe we should consider for another task: The original compliance status has an endpoint `getComplianceStatusList` to return a list of compliances for a list of consumers; do we need a similar endpoint for syspurpose compliance?